elastic/integrations
# [falco] Adjust Falco events to work with External Alerts rule

## Proposed commit message



Adjust some values in the falco ingest pipeline to work with the default 'External Alert' detection rule in Kibana. Doing this will make setting up detection alerts on ingested Falco events easier, since users can use the existing, builtin, detection rule.

Changes are:
* Set 'message' to `rule.name` - the rule uses `message` for the alert's display name
* Add host.name and user.name ECS mappings
* Set 'event.severity' to match alert severity override. The alert rule uses '=' comparator to set severity override, so the values must match exactly the new values set here for the alert to work.

The severity values are taken from the [External Alert rule definition](https://github.com/elastic/detection-rules/blob/main/rules/promotions/external_alerts.toml).

Falco priority meaning is defined [here](https://falco.org/docs/rules/basic-elements/#priority) and Elastic severity meaning is defined [here](https://www.elastic.co/guide/en/security/current/rules-ui-create.html#rule-ui-basic-params).

The mappings used in the integration are:
Falco CRITICAL or higher -> Elastic Critical
Falco ERROR -> Elastic High
Falco WARNING or NOTICE -> Elastic Medium
Falco INFORMATIONAL or lower -> Elastic Low

## Checklist



## How to test this PR locally



1. Install the integration
2. In Kibana, Security-Rules add and enable the 'External Alerts' detection rule.
3. Set up Falco and falcosidekick to send alerts to ES. See https://github.com/falcosecurity/falcosidekick/tree/master?tab=readme-ov-file#installation and https://github.com/falcosecurity/falcosidekick/blob/master/docs/outputs/elasticsearch.md
